Cannot read from file: '/home/runner/.config/mark' because it does not exist.

Today all the GitHub actions in different projects are failing with the error:

```
Run mark -u *** -p *** -b *** -f README.md
  mark -u *** -p *** -b *** -f README.md
  shell: /usr/bin/bash -e {0}
[2](https://github.com/Vonage/gosrvlib/actions/runs/4795215401/jobs/8529526393#step:5:2)02[3](https://github.com/Vonage/gosrvlib/actions/runs/4795215401/jobs/8529526393#step:5:3)-0[4](https://github.com/Vonage/gosrvlib/actions/runs/4795215401/jobs/8529526393#step:5:5)-2[5](https://github.com/Vonage/gosrvlib/actions/runs/4795215401/jobs/8529526393#step:5:6) 08:16:46.817 FATAL Unable to create input source with context: inner error: 
                              'Unable to load TOML file '/home/runner/.config/mark': inner error: 
                              'Cannot read from file: '/home/runner/.config/mark' because it does not exist.''
Error: Process completed with exit code 1.
```

The workflow configuration can be seen at:
https://github.com/Vonage/gosrvlib/blob/main/.github/workflows/confluence.yaml

I am not sure if this is the result of a breaking change? or is it a new bug?

--- a/main.go
+++ b/main.go
@@ -154,7 +154,11 @@ func main() {
 					filePath := context.String("config")
 					return altsrc.NewTomlSourceFromFile(filePath)
 				} else {
-					// Fall back to default if config is unset
+					// Fall back to default if config is unset and path exists
+					_, err := os.Stat(filepath.Join(os.Getenv("HOME"), ".config/mark"))
+					if os.IsNotExist(err) {
+						return &altsrc.MapInputSource{}, nil
+					}
 					return altsrc.NewTomlSourceFromFile(filepath.Join(os.Getenv("HOME"), ".config/mark"))
 				}
 			}),

